docs/platform/howto/byoc/create-custom-cloud/create-aws-custom-cloud.md

```diff
@@ -580,24 +464,39 @@ In the **Create custom cloud** wizard:
             cannot change the BYOC VPC CIDR block after your custom
             cloud is created.
 
-    -   Deployment model: Choose between
-        [the private architecture and the public architecture](/docs/platform/concepts/byoc).
+    -   [Deployment model](/docs/platform/concepts/byoc#byoc-architecture)
+
+        Choose between:
+        - Private model, which routes traffic through a proxy for additional security
+          utilizing a bastion host physically separated from the Aiven services.
```

Comment:
I wouldn't say it's physically separated. We don't really know where the actual virtual machines are running, they might be in the same host. However, they are logically separated as they are in different subnets of the VPC